test/command/cmd.go

```diff
@@ -213,7 +215,9 @@ func (s *Starter) KillAll() {
 func (s *Starter) Clean() {
 	s.KillAll()
 	if s.Home != "" {
-		os.RemoveAll(s.Home)
+		if err := os.RemoveAll(s.Home); err != nil {
```

Comment:
I do not think every error should be declared and printed. Even printed, not necessary as a Error level.
You can see lots of docs, like:
https://travix.io/unhandled-errors-in-go-3f341f2704dd

Reply:
Sure.I will think about that more.what is your  advice? ignore or  lower level?

Reply:
thanks your sharings